src/internal/connector/onedrive/collections.go

```diff
@@ -325,6 +328,33 @@ func (c *Collections) Get(
 		folderPaths[driveID] = map[string]string{}
 		maps.Copy(folderPaths[driveID], paths)
 
+		// Set all folders in previous backup but not in the current
+		// one with DoNotMerge
+		if c.ctrl.ToggleFeatures.EnableOneDriveDeltaIncrementals && delta.Reset {
+			for i, p := range oldPaths {
+				_, found := paths[i]
+				if !found {
+					prevPath, err := path.FromDataLayerPath(p, false)
+					if err != nil {
+						return nil, map[string]struct{}{}, clues.Wrap(err, fmt.Sprintf("invalid previous path for deleted item %s", p))
```

Comment:
logging standards: keep messages short and simple, don't fmt data.  Also, since the erroring function didn't take in a `ctx`, you'll want to plug in the clues from the context (WithClues(ctx)).
```suggestion
						return nil, map[string]struct{}{}, clues.Wrap(err, "invalid previous path").WithClues(ctx).With("deleted_path", p)
```